### PR TITLE
Split outputs

### LINKED ISSUE #5: Add g shortcuts
It's useful for coreutils to have g shortcuts, like `gsort`, and so on, so that scripts can ensure that they are using the coreutils versions of a command.

### PR DESCRIPTION
**Pull request by ngam**
Fixes #5 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ngam** [comment]
@dopplershift @ocefpaf (mainly because you both helped me a lot) or anyone in @conda-forge/core, could you please quickly double check on splitting this recipe to address issue #5? Thanks!

**Comment by ngam** [comment]
@conda-forge-admin, please rerender

**Comment by ngam** [comment]
I tested it locally and everything is fine. However, the tests don't run for coreutils if I split it this way for some reason... and I am not sure how to fix that.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,16 +14,36 @@ build:
...
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - make
+
+    test:
+      commands:
+        - gln --relative --help
+        - gwc --version
+        - gcat --version
```
Comment (asmeurer) [inline recipe/meta.yaml:48]: Can we add a test that `ln` is not coming from this package?

**Comment by asmeurer** [comment]
> I tested it locally and everything is fine. However, the tests don't run for coreutils if I split it this way for some reason... and I am not sure how to fix that.

Possibly a bug in conda build?

File: recipe/meta.yaml (hunk 1 above)
Comment (ngam) [reply recipe/meta.yaml:48]: Sure, fwiw I tested exactly that locally 😄

**Comment by ngam** [comment]
@isuruf 


Oh never mind! Thanks a lot

File: recipe/meta.yaml (hunk 1 above)
Comment (ngam) [reply recipe/meta.yaml:48]:
```

GNU coreutils online help: <https://www.gnu.org/software/coreutils/>
Full documentation <https://www.gnu.org/software/coreutils/ln>
or available locally via: info '(coreutils) ln invocation'
ln (GNU coreutils) 8.22
Copyright (C) 2013 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Mike Parker and David MacKenzie.
ln (GNU coreutils) 9.0
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

```

**Comment by ngam** [comment]
> > I tested it locally and everything is fine. However, the tests don't run for coreutils if I split it this way for some reason... and I am not sure how to fix that.
> 
> Possibly a bug in conda build?

fixed in 5785ff9 by isuruf

**Comment by ngam** [comment]
@asmeurer could you check if everything is working correctly on your end when you get a chance? We now have the two outputs and soon we will have them in all arches (arm, ppc, aarch) except windows. Let me know if anything could be fixed --- also, happy to add you as maintainer here if you wish

**Comment by asmeurer** [comment]
Seems to work for me (Intel Mac).

**Pull request merged**